src/nvim/os/fs.c

```diff
@@ -517,6 +521,8 @@ ptrdiff_t os_write(const int fd, const char *const buf, const size_t size)
     if (cur_written_bytes < 0) {
 #ifdef HAVE_UV_TRANSLATE_SYS_ERROR
       const int error = uv_translate_sys_error(errno);
+#elif WIN32
+      const int error = win32_translate_sys_error(errno);
```

Comment:
Might as well put these calls (`uv_translate_sys_error` and `win32_translate_sys_error`) behind `os_translate_sys_error` ?